client/scraper/youtube_metadata.py

```diff
@@ -32,7 +32,7 @@ def parse_content(self, response):
         media_list = {}
         media_list[FieldKeyword.DATA] = [
             {
-                FieldKeyword.URL: None,
+                FieldKeyword.URL: response.request_url,
```

Comment:
Does this refer to the oembed link? Or the regular sanitized url? Cause if it's the latter, it should just live in the prop_map no?